src/content/docs/synthetics/synthetic-monitoring/using-monitors/device-emulation.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: Device emulation
+tags:
+  - Synthetics
+  - Synthetic monitoring
+  - Using monitors
+translate:
+  - jp
+metaDescription: Emulating mobile or tablet devices in browser based synthetic monitors
+redirects:
```

Comment:
@bpeckNR you can remove this since this is a new doc and has no redirects yet.

Reply:
Done